### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Danfall369**
-Implement of object array in a javascript file to make a dynamic grid of project sections and a dynamic popup window.
-adding a popup window when the button 'see project' on the projects card is pressed.
-adding assets to make the design of the popup window.
-implement functions on the js file
-clean and order coder
-all lines checked

### CODE REVIEW

File: js/projectcode.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    return gridBox;
+  });
+};
+cards();
+
+// PopUP menu
+const popUp = document.querySelector('.detail-mobile');
+const section = document.createElement('div');
+popUp.append(section);
+section.innerHTML = `
```
Comment (topeogunleye) [inline js/projectcode.js:106]: - [ ] I kindly suggest that you follow you Figma design by adding a background blur for the modals and ensuring that your modals are well centered as seen in the Figma design below:


| Your Modal  | Figma Modal |
| ------------- | ------------- |
| image  | Screenshot 2023-03-09 194245  |

| Your Modal With No Background Blur | Figma Modal |
| ------------- | ------------- |
| image  | image |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Danfall369, @VelzckC0D3 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good description :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/projectcode.js (hunk 1 above)
Comment (Danfall369) [reply js/projectcode.js:106]: Done, I correct all the issues

Comment (topeogunleye) [reply js/projectcode.js:106]: > Done, I correct all the issues

@Danfall369 That's Great. Kindly submit for review pls

File: Style/style.css [hunk 2]
```diff
@@ -520,6 +433,217 @@ main {
...
+  top: 0;
+  border-radius: 0;
+  background: #c1c7d0;
+  transition: 200ms ease-in-out;
+  overflow-y: scroll;
+  background-color: rgba(0, 0, 0, 0.5);
+  backdrop-filter: blur(5px);
+}
+
+.card-works {
```
Comment (huseyinbiyik) [inline Style/style.css:450]: Good job with the background blur 👍  But let's fix the other request from the previous reviewer and align the modal to the center. As you can see below there are more space on the right for desktop and mobile versions.
Screenshot 2023-03-09 at 10 19 07 PM
Screenshot 2023-03-09 at 10 19 23 PM

File: Style/style.css [hunk 3]
```diff
@@ -205,6 +206,7 @@ main {
   display: flex;
   align-items: center;
   justify-content: center;
+  cursor: pointer;
 }
 
 @media screen and (min-width: 768px) {
```
Comment (huseyinbiyik) [inline Style/style.css:212]: [OPTIONAL] I recommend you to follow the [best practices for HTML and CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and find the repeating codes and DRY them. You have repeating codes in your CSS file 👍 
Screenshot 2023-03-09 at 10 27 48 PM

File: Style/style.css (hunk 2 above)
Comment (Danfall369) [reply Style/style.css:450]: the margins on my popup window are correct and now are see on the center of the screen

**Review by franclobo — APPROVED** [review]
Hi @Danfall369 and @VelzckC0D3, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Dynamic webpage 💯 

## Optional suggestions
- **[OPTIONAL]** It doesn't make sense to have `@media screen and (min-width: 768px)` twice in the CSS file, please 🙏🏽 fix it for the best practices of HTML & CSS.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**